### ISSUE TITLE
Unable to resolve module 'util'

### ISSUE THREAD

**Issue by WrathChaos** [WoLfulus/zoya#2]
I tried to use zoya in React Native. I got the error: 

![Simulator Screen Shot - iPhone 11 - 2019-12-20 at 13 09 24](https://user-images.githubusercontent.com/6813862/71248145-3d770e80-232b-11ea-829a-6cc0c6de2e20.png)

Do you know any suggestion for that? I really want to use zoya from my mobile projects :)

**Comment by cspotcode** [comment]
@WrathChaos 
My guess is it depends on node's built-in `util` library, so it is not intended for use in React Native.  If React Native doesn't have an implementation of node's `util` library, I think you're out-of-luck.

**Comment by WoLfulus** [comment]
yeah, it works on node environments only right now. I'll take a look on supporting console and other environments too.

I'll leave this open until I find some time to think about it

**Comment by WrathChaos** [comment]
Waiting for it :) Thank you and awesome job
